i18n/de.json

```diff
@@ -51,5 +51,6 @@
 	"source-code": "Quellcode",
 	"login": "Anmelden",
 	"settings-link": "Einstellungen",
-	"settings-heading": "Einstellungen"
+	"settings-heading": "Einstellungen",
+	"logged-in": "{{GENDER:$2|Eongeloggt}} als $1."
```

Comment:
Typo, already fixed: https://translatewiki.net/w/i.php?title=Wikimedia:Wikidata-lexeme-forms-logged-in/de&diff=prev&oldid=11771036